sample-code/spring-app/src/main/java/com/sap/ai/sdk/app/controllers/OrchestrationController.java

```diff
@@ -41,148 +39,44 @@ class OrchestrationController {
-             Harvard University, Boston
-
-             Bachelor of Science: Finance - 2011
-             Harvard University, Boston
-
-
-             Certifications
-             Certified Management Accountant
-
-
-             Summary
-             Skilled Financial Manager adept at increasing work process efficiency and profitability through functional and technical analysis. Successful at advising large corporations, small businesses, and individual clients. Areas of expertise include asset allocation, investment strategy, and risk management.
-
-
-             Experience
-             Finance Manager - 09/2016 to 05/2018
-             M&K Group, York
-             - Manage the modelling, planning, and execution of all financial processes.
-             - Carry short and long-term custom comprehensive financial strategies to reach company goals.
-             - Recommended innovative alternatives to generate revenue and reduce unnecessary costs.
-             - Employed advanced deal analysis, including hands-on negotiations with potential investors.
-             - Research market trends and surveys and use information to stimulate business.
-
-             Finance Manager - 09/2013 to 05/2016
-             Ago Group, Chicago
-             - Drafted executive analysis reports highlighting business issues, potential risks, and profit opportunities.
-             - Recommended innovative alternatives to generate revenue and reduce unnecessary costs.
-             - Employed advanced deal analysis, including hands-on negotiations with potential investors.
-             - Analysed market trends and surveys and used information to revenue growth.""");
-            final var template =
-                ChatMessage.create()
-                    .role("user")
-                    .content("Summarize the following CV in 10 sentences: {{?orgCV}}");
-            final var templatingConfig = TemplatingModuleConfig.create().template(template);
-
-            final var maskingProvider =
-                MaskingProviderConfig.create()
-                    .type(MaskingProviderConfig.TypeEnum.SAP_DATA_PRIVACY_INTEGRATION)
-                    .method(MaskingProviderConfig.MethodEnum.ANONYMIZATION)
-                    .entities(ALL_DPI_ENTITIES);
-            final var maskingConfig =
-                MaskingModuleConfig.create().maskingProviders(maskingProvider);
-
-            return CompletionPostRequest.create()
-                .orchestrationConfig(
-                    OrchestrationConfig.create()
-                        .moduleConfigurations(
-                            ModuleConfigs.create()
-                                .llmModuleConfig(LLM_CONFIG)
-                                .templatingModuleConfig(templatingConfig)
-                                .maskingModuleConfig(maskingConfig)))
-                .inputParams(inputParams);
-          };
+  @GetMapping("/completion")
+  @Nonnull
+  public CompletionPostResponse completion() {
```

Comment:
Added a simple completion, since templating is optional (and potentially easier to do in other places of the code).